flow-typed/npm-custom/devtools-contextmenu_vx.x.x.js

```diff
@@ -0,0 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
+
+declare module "devtools-contextmenu" {
```

Comment:
An idea would be following this up with publishing these to `flow-typed`, but let's add them here first and flesh them out.